### PR TITLE
Add ignored patches feature to PaStA

### PR DESCRIPTION
**Pull request by 8sd**
This pull request adds a feature to detect ignored patches.
In addition, the pull request contains a feature to export the characteristics of the patches analyzed.

### CODE REVIEW

**Review by rralf — CHANGES_REQUESTED** [review]
Could you please squash those changes to the patch "Add script to evaluate patches"?

**Comment by rralf** [comment]
... And please rebase to latest next. Otherwise, things look fine. Thanks!

**Comment by 8sd** [comment]
@rralf I did both tasks

**Comment by rralf** [comment]
Thanks, applied.

File: bin/pasta_evaluate_patches.py [hunk 1]
```diff
@@ -2,13 +2,319 @@
...
+                   'time': c.date,
+                   'mtrs_correct': mtrs_correct}
+
+            writer.writerow(row)
+
+
 def evaluate_patches(config, prog, argv):
+<<<<<<< HEAD
     pass
+=======
```
Comment (rralf) [inline bin/pasta_evaluate_patches.py:248]: Huh? How did that happen? Will remove this chunk and rewrite next.

**Pull request merged**